config/locales/en.yml

```diff
@@ -30,6 +30,30 @@ en:
       sign_up: Sign Up
       sign_in: Sign In
       sign_out: Sign Out
+  layouts:
+    global_announcements:
+      close: Close
+      more_announcements: More Announcements...
+  admin:
+    navbar:
+      announcements: Announcements
+    admin:
+      home:
+        message: "Pages"
+    announcements:
+      index:
+        header: "Announcements"
```

Comment:
Hmmm... http://stackoverflow.com/a/22235064 says

> Use quotes if your value includes special characters, (e.g. :, {, }, [, ], ,, &, *, #, ?, |, -, <, >, =, !, %, @, \).
> "Yes" and "No" should be enclosed in quotes (single or double) or else they will be interpreted as TrueClass and FalseClass values:

Since I expect `#{...}` to be very common, let's stick to using quotes. How about single quotes? Since we are already preferring single quotes in ruby.